### ISSUE TITLE
Make the installation docs less dependent of GitHub pages

### ISSUE THREAD

**Issue by niconoe** [peterdesmet/petridish#84]
I am trying to experiment with Petridish but it seems, at least from the docs that it is rather tied to GitHub pages. I struggled a bit with this: the recommended method assumes it's for a `Project site` and doesn't exactly match the experience with a user/organization site. Also, while GitHub pages is convenient, I think there might be a lot of good reasons we want to host it elsewhere.

I therefore wanted to first develop a local site on my machine, and think about the hosting later. I was not able to use the third suggested method: `Install Petridish as gem-based theme`. But if I am not mistaken, the gem is not published yet: https://rubygems.org/search?query=petridish

Note that there's also a typo (what -> want?) in the following sentence: "If you do not what to use remote_theme, see the [Jekyll documentation](https://jekyllrb.com/docs/themes/#understanding-gem-based-themes) to install (gem-based) themes."

This also references the `remote_theme` settings, which is also GitHub pages-specific, I think.

It would be great if the documentation showed a bit more clearly that Petridish-based sites can be conveniently hosted on GitHub pages, but that it is also a standard Jekyll theme that can be used (and how?) whatever your hosting choices are. Maybe a few small changes to the documentation + automatic pushes on rubygems for each release would be enough?

I'm volunteering to be a guinea pig if useful since I have a site to build, but not urgently :)

PS: I am not really familiar with Jekyll, maybe that's why I can't really connect the dots by myself.

**Comment by niconoe** [comment]
Indeed, I think a first useful step so I can help testing is to actually publish the theme at rubygems. 

I would proceed like that:

- read generic instructions about [creating a gem](https://guides.rubygems.org/make-your-own-gem/) and [publishing a gem](https://guides.rubygems.org/publishing/) at rubygems
- check [Jekyll-specific considerations](https://jekyllrb.com/docs/themes/#creating-a-gem-based-theme)
- apply this new knowledge and publish it :)
- we can then do a quick test from the user point of view ([follow Jekyll docs to use a theme from a gem](https://jekyllrb.com/docs/themes/#installing-a-theme)) and check for obvious issues
- if all goes well, I can start using it on my site and report issues if needed
- then, I suggest to automate the "publishing to rubygems" part (maybe with GitHub actions, each time a GitHub release is created)

I guess it's better if do the publishing part yourself, so you appear as the gem owner/maintainer. I don't have experience with the process, but if you'd like to work with 4 hands on this, we can arrange a session (remotely or at VAC) for that.

**Comment by peterdesmet** [comment]
The theme is now published at https://rubygems.org/gems/jekyll-theme-petridish

Since `petridish` was to similar to an already existing gem `petri_dish`, I had to name it `jekyll-theme-petridish` (following what seems the most common naming convention). Once you have installed it:

```
gem install jekyll-theme-petridish
```

You can refer to it in your `_config.yml` with:

```
theme: jekyll-theme-petridish
```

The theme might include a bit too much clutter (incl. `_data` files) at the moment. Will create a separate issue for that, but let me know what you find useful on a fresh install.
